admin/tool/policy/classes/api.php

```diff
@@ -572,9 +555,9 @@ public static function get_user_acceptances($userid, $versions = null) {
             $vsql = ' AND a.policyversionid ' . $vsql;
         }
 
-        $sql = "SELECT u.id AS mainuserid, a.*, u.policyagreed
-                  FROM {user} u
-                  LEFT OUTER JOIN {tool_policy_acceptances} a ON a.userid = u.id AND a.userid = :userid $vsql";
+        $sql = "SELECT u.id AS mainuserid, a.policyversionid, a.status, a.language, u.policyagreed
+                  FROM {user} u, {tool_policy_acceptances} a
```

Comment:
While this is valid SQL, it is generally preferred to use "JOIN" (or even "INNER JOIN") variant instead of this shortened comma separated list of tables with the binding condition in WHERE.

By the way, is it expected that now, as you are changing the left outer join to the inner join, you will no longer get users without an acceptance record?

Reply:
Thanks for your helpful comments. :-* 

I'll change this SQL to INNER JOIN. In this case, the behaviour in both cases (the WHERE clause and the INNER JOIN) is the same, so only are returned the acceptances records for the defined userid.